icu4c/source/test/intltest/ustrtest.cpp

```diff
@@ -22,6 +22,42 @@
 #include "cmemory.h"
 #include "charstr.h"
 
+
+extern void tempCompileTest() {
```

Comment:
FYI just to make sure that the sv16 & str16 don't work before adding the overloads, and then make sure that they at least compile. This will eventually turn into real test code that actually looks at the objects.